src/components/Popups/PopupBase.scss

```diff
@@ -11,6 +11,12 @@
 
         z-index: 1;
```

Comment:
Maybe also add a comment here explaining why this is needed. I should have done so when I added it.